docs/data/tree-view/rich-tree-view/focus/focus.md

```diff
@@ -0,0 +1,24 @@
+---
+productId: x-tree-view
+title: Rich Tree View - Focus
+components: RichTreeView, TreeItem
+packageName: '@mui/x-tree-view'
+githubLabel: 'component: tree view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+---
+
+# Rich Tree View - Focus
+
+<p class="description">Learn how to focus Tree View items.</p>
+
+## Focus a specific node
+
+You can access the `focusNode` method using the `apiRef` variable. To access this variable, use the `useTreeViewApiRef` hook.
```

Comment:
I propose to put the `apiRef` init example in a `success`, duplicate it for the 2-3 first methods we document in the doc and then see to create a standalone section with a proper listing of the supported methods like on the grid.

Something like:

````suggestion
You can use the the `apiRef.focusNode` method to focus a specific node.
This methods receives two parameters: `event` and `nodeId`.

:::success
To use the `apiRef` object, you need to initialize it using the `useTreeViewApiRef` hook as follows:

```tsx
const apiRef = useTreeViewApiRef();

return (
  <RichTreeView apiRef={apiRef}>
    {children}
  </RichTreeView>
);
```

`apiRef` will be undefined during the first render and will then contain methods allowing you to imperatively interact with the Tree View.
:::
````